scripts/test-unix.sh

```diff
@@ -54,6 +54,13 @@ else
   echo "Server started in $((RC_START_TIMEOUT_IN_SECS - timeout)) seconds"
 fi
 
+if [ "${BUILD_TYPE}" = "Debug" ]; then
```

Comment:
```suggestion
if [ "${BUILD_TYPE}" = "Debug" AND "x${ASAN_OPTIONS}" = "x" ]; then
```
only if we did not externally provide the options, then disable it so that we can run the script to fix the leaks.

Reply:
Good point

4ab2a535e564c9878395eb2ecedeb879fbda89e0